examples/specs/bubble_health_income.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "A bubble plot showing the correlation between health and income for 187 countries in the world.",
```

Comment:
Still, we modified Lisa's code and even copied data from her copy.  Definitely doesn't hurt to credit her and her article.    In fact, it might be nice to ask her for a permission to copy.